audit/projects/k8s-staging-csi/services/enabled.txt

```diff
@@ -8,8 +8,11 @@ clouddebugger.googleapis.com      Cloud Debugger API
 cloudkms.googleapis.com           Cloud Key Management Service (KMS) API
 cloudtrace.googleapis.com         Cloud Trace API
 compute.googleapis.com            Compute Engine API
+container.googleapis.com          Kubernetes Engine API
```

Comment:
This was @dims.  It's sadly easy to enable just by clicking the UI wrong.  Not harmful, but I manually reverted this anyway.

We need  Hourly runs of audit and PRs :)

Reply:
whoa! Sorry again